tests/test_app.py

```diff
@@ -0,0 +1,69 @@
+import os
+import requests
+import pytest
+from dotenv import load_dotenv, find_dotenv
+from todo_app import app
+
+@pytest.fixture
+def client():
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+    test_app = app.create_app()
+
+    with test_app.test_client() as client:
+        yield client
+
+def test_index_page_loads(monkeypatch, client):
+    
+    # Given
+    monkeypatch.setattr(requests, 'get', get_lists_stub)
+
+    # When
+    response = client.get('/')
+
+    # Then
+    assert response.status_code == 200
+    assert 'Just another to-do app.' in response.data.decode()
+
+def test_index_page_displays_trello_data(monkeypatch, client):
+    
+    # Given
+    monkeypatch.setattr(requests, 'get', get_lists_stub)
+
+    # When
+    response = client.get('/')
+
+    # Then
+    assert 'Test card' in response.data.decode()
```

Comment:
You could make this test stronger by specifying where 'Test card' string appears. For now it will return success when you get a message like "Could not find Test card", etc. It's clear that it won't happen here right now, but you will be forever safe if you specify where in JSON it appears.